Makefile

```diff
@@ -132,12 +133,21 @@ bin/golangci-lint:
 	mkdir -p bin
 	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | BINARY=golangci-lint bash -s -- v${GOLANGCI_VERSION}
 
-.PHONY: lint lint-fix
+bin/gofumpt:
+	mkdir -p bin
+	test -f bin/gofumpt-$(GOFUMPT_VERSION) || curl -sLo bin/gofumpt-$(GOFUMPT_VERSION) https://github.com/mvdan/gofumpt/releases/download/v$(GOFUMPT_VERSION)/gofumpt_v$(GOFUMPT_VERSION)_$(OS_NAME)_$(PLATFORM_NAME)
+	chmod u+x bin/gofumpt-$(GOFUMPT_VERSION)
+	ln -s gofumpt-$(GOFUMPT_VERSION) bin/gofumpt
+
+.PHONY: lint lint-fix format
 lint: ## Run linter.
-	golangci-lint run
+	bin/golangci-lint run
```

Comment:
NIT: I assume that $PWD/bin is in the $PATH, so you don't need to add it here, yet I could be mistaken.

Reply:
I don't have $PWD/bin in my PATH. I don't expect it to work. I noticed that this Makefile called golangci-lint installed by Homebrew, and it has different version

Reply:
https://github.com/deckhouse/deckhouse/blob/6c008d46b45ec6df7a67eedb9ca17f0e13a2f889/Makefile#L1